src/engraving/layout/v0/chordlayout.cpp

```diff
@@ -2288,12 +2293,12 @@ void ChordLayout::layoutChords3(const MStyle& style, const std::vector<Chord*>&
                 // resolve dot conflicts
                 int line = note->line();
                 Note* above = (i < nNotes - 1) ? notes[i + 1] : 0;
-                if (above && (!above->visible() || above->dotsHidden())) {
+                if (above && (!above->visible() || above->dotsHidden() || !above->chord()->dots())) {
                     above = 0;
                 }
                 int intervalAbove = above ? line - above->line() : 1000;
                 Note* below = (i > 0) ? notes[i - 1] : 0;
-                if (below && (!below->visible() || below->dotsHidden())) {
+                if (below && (!below->visible() || below->dotsHidden() || !below->chord()->dots())) {
```

Comment:
You should not implicitly convert int to bool

Reply:
Fixed!